include/chopper/layout/hibf_statistics.hpp

```diff
@@ -99,8 +99,39 @@ class hibf_statistics
         expected_HIBF_query_cost = total_query_cost / total_kmer_count;
     }
 
+    //!\brief Prints a column names of the summary to the command line.
+    static void print_header(bool verbose = true)
+    {
+        // print column names explanation in header
+        std::cout << "## Note: X-IBF = An IBF with X number of bins.\n"
+                  << "## Note: X-HIBF = An HIBF with tmax = X, e.g a maximum of X technical bins on each level.\n";
+
+        std::cout << "## Column tmax : The maximum number of technical bin on each level\n"
+                     "## Column c_tmax : The technical extra cost of querying an tmax-IBF, compared to 64-IBF\n"
+                     "## Column l_tmax : The estimated query cost for an tmax-HIBF, compared to an 64-HIBF\n"
+                     "## Column m_tmax : The estimated memory consumption for an tmax-HIBF, compared to an 64-HIBF\n"
+                     "## Column (l*m)_tmax : Computed by l_tmax * m_tmax\n"
+                     "## Column size : The expected total size of an tmax-HIBF\n"
+                  << ((verbose) ? "## Column uncorr_size : The expected size of an tmax-HIBF without FPR correction\n" : "");
```

Comment:
```suggestion
        std::cout << "## tmax : The maximum number of technical bins on each level\n"
                  << "## c_tmax : The query-runtime-ratio of a tmax-IBF to a 64-IBF\n"
                  << "## l_tmax : The IBF-query-calls-ratio of a tmax-HIBF to a 64-HIBF\n"
                  << "## m_tmax : The memory-consumption-ratio of a tmax-HIBF to a 64-HIBF\n"
                  << "## (l*m)_tmax : Product of time and memory ratios\n"
                  << "## size : The expected size of a tmax-HIBF\n"
                  << ((verbose) ? "## uncorr_size : The expected size of a tmax-HIBF without FPR correction\n" : "");
```

I don't think the `Column` prefix adds much?

Reply:
hm if you look at the example with a full header in the PR description, I added `Parameter, Note, Column` as prefix because I thought it makes it more distinguishable.

Reply:
It does, but it's also kinda noisy...
Maybe put the category as header?

```
## Parameters
## number of user bins = 8
## number of hash functions = 2
## false positive rate = 0.05
## Notes
## X-IBF = An IBF with X number of bins.
## X-HIBF = An HIBF with tmax = X, e.g a maximum of X technical bins on each level.
## Columns
## tmax : The maximum number of technical bin on each level
## c_tmax : The technical extra cost of querying an tmax-IBF, compared to 64-IBF
## l_tmax : The estimated query cost for an tmax-HIBF, compared to an 64-HIBF
## m_tmax : The estimated memory consumption for an tmax-HIBF, compared to an 64-HIBF
## (l*m)_tmax : Computed by l_tmax * m_tmax
## size : The expected total size of an tmax-HIBF
```

Reply:
That's nice!